### DISCUSSION TITLE
Would it make sense to install dbmate using Go?

### DISCUSSION THREAD

**Discussion by x80486** [amacneil/dbmate#463] [Support] [answered]
I've been going through the documentation, specifically the `Installation` section. I don't see that it's recommended, or even listed, to install `dbmate` using `go install` — e.g.: `go install github.com/amacneil/dbmate/v2@v2.5.0`.

Are there any drawbacks with this approach?

I'm fairly new to `Go`, that's why I may not fully understand the implications, but I can see `dbmate` is available using that method. I'm just thinking to use this approach for local development; for other purposes (e.g.: CI/CD setup) the binaries are better.

**Comment by amacneil** [comment, marked as answer]
Yes, that is possible. The drawbacks are that it requires a local Go toolchain, and you have to wait for it to download all dependencies and compile. 

That's the only reason we recommend the precompiled binary as the primary installation method.
